### ISSUE TITLE
wikipedia error message displayed.

### ISSUE THREAD

**Issue by mattlub** [facn2/ApolloXI#59]
e.g. when I searched for 'barack obama'

the wikipedia info displayed was "From a miscapitalisation: This is a redirect from a miscapitalisation. The correct form is given by the target of the redirect..."

**Comment by veredrec** [comment]
@mattlub You are totally right. The Wikipedia API that we worked with, only accepts exact words (not as YouTube) so if you type "Obama" you will get the article, but if not, you won't get any useful data...

**Comment by mattlub** [comment]
is there any way to remedy this?

**Comment by rinoma** [comment]
Nope, looked long and wide and couldn't find an appropriate solution

**Comment by rinoma** [comment]
@mattlub close please
